src/Components/rockets/Rockets.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getRockets } from '../../features/rocketsSlice';
+import Rocket from './Rocket';
+import '../../Assets/css/rockets.css';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const { rockets } = useSelector((store) => store.rocket);
+  useEffect(() => {
+    dispatch(getRockets());
+  }, [dispatch]);
```

Comment:
It will be great to ensure that the data is not stored in the store with every render. What is happening now in your application is that if I navigate to another page the data is fetched again. This should only be done once and then every other time it should get its data from the store. Not from the API.
⬇️
![data store](https://user-images.githubusercontent.com/32361963/235104029-4b7847d8-aad8-47da-8a00-cfb97c5af073.png)